server/app/serve_index.go

```diff
@@ -37,6 +39,7 @@ func serveIndex(ds model.DataStore, fs fs.FS) http.HandlerFunc {
 			"version":                 consts.Version(),
 			"firstTime":               firstTime,
 			"baseURL":                 policy.Sanitize(strings.TrimSuffix(conf.Server.BaseURL, "/")),
+			"auth":                    payload,
```

Comment:
One mor comment: This now appears in debug logs. It should be redacted. Ex:
```
DEBU[0004] UI configuration                              appConfig="map[auth:map[id:dfd95934-35fb-41e1-801e-710f6dafd9bd isAdmin:true message:User 'admin' authenticated successfully name:Dev Admin subsonicSalt:f6e4ae subsonicToken:c52d8017ba5b8791847e27785ad7a4af token:eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZG0iOnRydWUsImV4cCI6MTYyMzE5MjEwMywiaWF0IjoxNjIzMTA1NzAzLCJpc3MiOiJORCIsInN1YiI6ImFkbWluIiwidWlkIjoiZGZkOTU5MzQtMzVmYi00MWUxLTgwMWUtNzEwZjZkYWZkOWJkIn0.8C7Pv-rwcWWBYWF7rORaxPb41r_ssTLiliiq_Rt7SYI username:admin] baseURL: defaultTheme:Dark devActivityPanel:true devEnableShare:false devFastAccessCoverArt:true enableDownloads:true enableFavourites:true enableStarRating:true enableTranscodingConfig:false enableUserEditing:false firstTime:false gaTrackingId: loginBackgroundURL:https://source.unsplash.com/collection/20072696/1600x900 losslessFormats:FLAC,WAV,ALAC,APE,DSF,WAV,SHN,WV,WVP version:dev welcomeMessage:Long long long long String <br/>Another String]"
```

You can find the redaction list here: https://github.com/navidrome/navidrome/blob/39d68e8287bc68044b5f9ad8f12d7137ee7a56a6/log/log.go#L21

Reply:
I added regexes to `log/log.go` but creds are still not redacted - am I doing something wrong? I even tested those regexes manually, and they should catch proper strings.

Reply:
I'll take a look later tonight. Maybe it is a limitation of the redacting code

Reply:
It was an issue with the redactor. I fixed it and pushed to your branch.